src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -17,6 +17,7 @@ public EndpointParameter(Endpoint endpoint, IParameterSymbol parameter, WellKnow
     {
         Type = parameter.Type;
         Name = parameter.Name;
```

Comment:
`Name` -> `ParameterName`

Makes the difference between the two names obvious.

Reply:
May other PR submitters forgive us....

Reply:
I guess that `ParameterName` is still not clear. `ParameterSymbolName`?

Reply:
`SymbolName`!

Reply:
I think ParameterName is fine. I was pointing out the fact that will be fun to anyone with an inflight PR because of changing that field name. It wasn't too bad though. Heads up for @captainsafia.